sponsorship/src/main/java/com/kyriba/conference/sponsorship/domain/Plan.java

```diff
@@ -5,18 +5,42 @@
 import lombok.Data;
 import lombok.NoArgsConstructor;
 
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.SequenceGenerator;
+import javax.persistence.Table;
+
 
 /**
  * @author M-ASL
  * @since v1.0
  */
+@Entity
+@Table(name = "plan")
 @Data

```

Comment:
following the comment in another PR, an entity class shouldn't be marked with Data annotation

Reply:
Agree, Data generates equals and hashCode that cause many surprises for persistent entities.